src/main/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStatic.java

```diff
@@ -245,6 +257,21 @@ private static List<J.Try.Resource> getMatchingFilteredResources(@Nullable List<
         return emptyList();
     }
 
+    private static Optional<J.Identifier> tryGetMatchedWrappingVariableName(Cursor cursor, String className) {
+        return VariableUtil.findVariablesInScope(cursor).stream()
+                .filter(identifier -> {
+                    if (MOCKED_STATIC.matches(identifier) && identifier.getType() instanceof JavaType.Parameterized) {
+                        JavaType.Parameterized parameterizedType = (JavaType.Parameterized) identifier.getType();
+                        if (parameterizedType.getTypeParameters().size() == 1) {
+                            JavaType mockedClass = parameterizedType.getTypeParameters().get(0);
+                            return TypeUtils.isAssignableTo(className, mockedClass);
```

Comment:
```suggestion
                            return TypeUtils.isAssignableTo(className, parameterizedType.getTypeParameters().get(0));
```

Optional change, just shorter (but arguably less readable).

Reply:
Agreed, I think this one was was still from checking out what was available / working during debugging session.